iroh-net/src/magicsock.rs

```diff
@@ -1002,7 +1002,7 @@ impl Inner {
                 actor_sender
                     .send(ActorMessage::ReStun("refresh-for-peering"))
                     .await
-                    .unwrap();
+                    .ok()
```

Comment:
Still would like to turn this into a try_send if possible, but that can happen later? Maybe a todo: investigate if try_send is ok here?

Reply:
Yes I have a followup already that changes this. Will push to a new PR.